docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,57 @@
+login
+create-account
+    - implicit
+    - subaccount
+manage-account
+    <!-- Should we add local key management? -->
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - contract
+        - deploy
+        - dev-deploy
+        - state
+        - code
+        <!-- TODO: what is the best name for it? -->
+        - change-function-call
+        <!-- TODO: what is the best name for it? -->
+        - view-function-call
+    - state
+    - delete
+transfer
+<!-- TODO: should it be manage-account? Should it be here at all? If yes, where is `unstake`? Maybe it should be an extension, like "near-cli-extension-staking".  -->
+stake
+<!-- TODO: feels a bit out of place, but definitely important. Maybe we need to add other view functions and have 1 subcommand for them. -->
+transaction-status
+helpers
+    <!-- TODO: some or all of these helpers can live in extensions, let's discuss it -->
```

Comment:
yea I think all of these can live in extensions. But things like `contract-transaction` and `recent-block-hash` should already be installed by default since those seem very essential